docker-entrypoint.sh

```diff
@@ -80,7 +80,16 @@ if expr "$*" : '^java .*/start\.jar.*$' >/dev/null ; then
 		set -- $(cat /jetty-start)
 	else
 		# Do a jetty dry run to set the final command
-		set -- $("$@" --dry-run | sed 's/\\$//' )
+		"$@" --dry-run > /jetty-start
+		EXIT=$?
```

Comment:
It's good practice to keep `set -e` globally in case we introduce any bugs later where we're not properly accounting for failed commands.